bigtop-packages/src/common/hadoop/do-component-build

```diff
@@ -120,6 +120,12 @@ if [ -z "$BUNDLE_SNAPPY" ] ; then
   [ -f /usr/lib/${HOSTTYPE}-linux-gnu/libsnappy.so ] && BUNDLE_SNAPPY="-Dsnappy.prefix=x -Dbundle.snappy=true -Dsnappy.lib=/usr/lib/${HOSTTYPE}-linux-gnu"
 fi
 
+if [ -z "$BUNDLE_ZSTD" ] ; then
+  [ -f /usr/lib/libzstd.so ] && BUNDLE_ZSTD="-Dbundle.zstd=true -Dzstd.lib=/usr/lib"
+  [ -f /usr/lib64/libzstd.so ] && BUNDLE_ZSTD="-Dbundle.zstd=true -Dzstd.lib=/usr/lib64"
+  [ -f /usr/lib/${HOSTTYPE}-linux-gnu/libzstd.so ] && BUNDLE_ZSTD="-Dbundle.zstd=true -Dzstd.lib=/usr/lib/${HOSTTYPE}-linux-gnu"
```

Comment:
Can we expect all supported distro/platform have libzstd.so? We can add package dependency on libzstd to make it certain that the required library is installed to runtime environment if so.

Reply:
emmm, I only have CentOS 7.4 at hand.
Can someone help to verify other distros/platforms?

Reply:
> emmm, I only have CentOS 7.4 at hand. Can someone help to verify other distros/platforms?

Assume your development environment is x86 64/amd64-based.
Please try to test it in various Distros of Bigtop docker images for x86 64/amd64 first.
(https://hub.docker.com/r/bigtop/slaves/tags)

Reply:
Ok, I'll paste the result later, thanks.

Reply:
With bigtop docker images for x86_64, got following result:
1. centos-7
<img width="623" alt="image" src="https://user-images.githubusercontent.com/12960246/231501093-9e201bce-1f7d-4380-bd21-3518bde9e963.png">
<img width="850" alt="image" src="https://user-images.githubusercontent.com/12960246/231501127-0c7b868b-b0d8-41a5-b0af-521ae57a05bf.png">
2. debian-11
<img width="709" alt="image" src="https://user-images.githubusercontent.com/12960246/231501184-cac4f922-3a94-4da7-ac2a-6e03555ed6d8.png">
<img width="1105" alt="image" src="https://user-images.githubusercontent.com/12960246/231501206-906cdf75-cc64-4332-ab23-676de5b45d00.png">
3. ubuntu-22.04
<img width="1388" alt="image" src="https://user-images.githubusercontent.com/12960246/231501252-314a31ff-8b5f-45f9-b4cf-8fd617890882.png">
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/12960246/231501275-ec2d2bc4-da91-4659-a34b-7c67bcc537c2.png">
4. fedora-36
<img width="770" alt="image" src="https://user-images.githubusercontent.com/12960246/231501304-0dce3785-5767-46dd-ad91-e9c8938db00b.png">
<img width="802" alt="image" src="https://user-images.githubusercontent.com/12960246/231501330-59355cc1-66e7-46ef-a01d-d3f839fec573.png">
5. rockylinux-8
<img width="933" alt="image" src="https://user-images.githubusercontent.com/12960246/231501393-cffcef2c-7c85-46c5-b6c3-597c86ca4576.png">
<img width="712" alt="image" src="https://user-images.githubusercontent.com/12960246/231501436-cd78df21-d521-4c24-9663-daffc3f53d68.png">